cmd/loop/main.go

```diff
@@ -176,26 +176,30 @@ func displayLimits(swapType swap.Type, amt btcutil.Amount, l *limits,
 		return nil
 	case "x":
 		fmt.Println()
-		if swapType != swap.TypeIn || !externalHtlc {
-			fmt.Printf("Max on-chain fee:                 %d\n",
-				l.maxMinerFee)
+		f := "%-36s %d sat\n"
+		if swapType == swap.TypeOut {
+			fmt.Printf(f, "Estimated on-chain sweep fee:",
+				minerFees)
+			fmt.Printf(f, "Max on-chain sweep fee:", l.maxMinerFee)
+		}
+		if swapType == swap.TypeIn && !externalHtlc {
```

Comment:
Could be a `switch` statement to prevent evaluation of this if when type is out.